modules/lang-groovy/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -25,6 +25,7 @@ grant {
   // needed by groovy engine
   permission java.lang.RuntimePermission "accessDeclaredMembers";
   permission java.lang.RuntimePermission "accessClassInPackage.sun.reflect";
+  permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
```

Comment:
@rmuir should look at this...there was an enormous amount of work that went into removing this permission across elasticsearch.